polaris/polaris/management/commands/execute_outgoing_transactions.py

```diff
@@ -81,7 +81,10 @@ def execute_outgoing_transactions():
         sep6_24_qparams = Q(
             protocol__in=[Transaction.PROTOCOL.sep24, Transaction.PROTOCOL.sep6],
             status=Transaction.STATUS.pending_anchor,
-            kind=Transaction.KIND.withdrawal,
+            kind__in=[
```

Comment:
Is there a particular reason to use double `_`?

Reply:
Its Django's ORM syntax. Its equivalent to the SQL `SELECT * WHERE kind IN ...`